docs/README.md

```diff
@@ -40,19 +40,15 @@ brew help
 make -h
 ```
 
-### Prerequisites: Fedora 29/Debian-based Linux Distributions
+### Fedora 29/Debian-based Linux Distributions
```

Comment:
I suggest dropping the number 29 (why 29?) and just say "Linux Distributions" - "Building the user docs should work out of the box on most Linux distributions".